bin/runtime/src/migrations/custom_nomination_pools.rs

```diff
@@ -0,0 +1,139 @@
+pub use nomination_pools::CustomMigrateToV2;
+
+mod nomination_pools {
+    use codec::{Decode, Encode, Error, Input};
+    use frame_support::{
+        log,
+        traits::{OnRuntimeUpgrade, StorageVersion},
+    };
+    use pallet_nomination_pools::{
+        BalanceOf, BondedPools, Config, Metadata, Pallet, PoolId, PoolMember, PoolMembers,
+        ReversePoolIdLookup, RewardPool, RewardPools, SubPoolsStorage,
+    };
+    use sp_core::{Get, U256};
+    use sp_std::{collections::btree_set::BTreeSet, prelude::*};
+
+    use crate::sp_api_hidden_includes_construct_runtime::hidden_include::dispatch::GetStorageVersion; // sick
+    use crate::Weight;
+
+    #[derive(Decode)]
+    pub struct OldRewardPool<B> {
+        pub balance: B,
+        pub total_earnings: B,
+        pub points: U256,
+    }
+
+    enum EitherRewardPool<T: Config, B> {
+        Old(OldRewardPool<B>),
+        New(RewardPool<T>),
```

Comment:
Is it correct that old is `16+16+32` bytes and new is `16+16+16` bytes?

Reply:
that is correct